platform/platform-paxweb-jettyconfig/src/main/java/org/codice/ddf/pax/web/jetty/JettyAuthenticatedUser.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.pax.web.jetty;
+
+import org.eclipse.jetty.server.Authentication;
+import org.eclipse.jetty.server.UserIdentity;
+
+public class JettyAuthenticatedUser implements Authentication.User {
+
+  private final UserIdentity userIdentity;
+
+  public JettyAuthenticatedUser(UserIdentity userIdentity) {
+    this.userIdentity = userIdentity;
+  }
+
+  @Override
+  public String getAuthMethod() {
+    return "DDF";
```

Comment:
❓ Does it make sense to use a common constant for this?

Reply:
This isn't used and can return null.